internal/convert.go

```diff
@@ -180,6 +181,11 @@ type streamingStats struct {
 	ShardToDataflowJobMap map[string]string
 }
 
+type dataprocStats struct {
```

Comment:
`dataProcStats` is misleading, probably `dataProcMetadata` or `dataProcIdMap` is a better naming here

Reply:
Done
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/520/commits/c27b72ed48a710da764d418d69f1f40ac30f5950